recipes/autogluon.common/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "autogluon.common" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/autogluon.common-{{ version }}.tar.gz
```

Comment:
If you're sdist is incomplete (that's probably a packaging bug) you can use the GitHub release source like below:
https://github.com/conda-forge/staged-recipes/blob/44060d157d267d0c8a5bd54f347edd098add72a7/recipes/example/meta.yaml#L17-L19

Reply:
The [Autogluon release](https://github.com/autogluon/autogluon/releases) on GitHub is the source code that contains all the subpackages of Autogluon, such as `autolugon.core`, `autolugon.vision` . `autogluon.common`'s `setup.py` is not under the root directory. Would that work for conda-forge?

![image](https://user-images.githubusercontent.com/5016453/212212715-ad0e9417-54ab-46f2-a124-aa6c86ca1fc1.png)

Reply:
Can we use the GitHub release and then have this line in the recipe?

```bash
script: {{ PYTHON }} -m pip install ./common -vv
```